### PR TITLE
Sap aem int day 3 - updated adapter import

### PR DESCRIPTION
**Pull request by ChristianHoltfurth**
Updated the adapter import instructions.

### CODE REVIEW

File: markdown/sap-aem-int-day-3/sap-aem-int-day-3.md [hunk 1]
```diff
@@ -516,8 +523,8 @@ Lastly, the AEM Receiver adapter is configured to persistently (to avoid message
 
 
 #### 2. Configuring and deploying the AEMBusinessPartnerAddressCheck iflow:
-![DQM service configuration](img/CIDQMServiceConfiguration.png)
-- Populate the connection details for the DQM service call out with the ones for your own DQM service instance.
+![DQM service configuration](img/CIDQMServiceConfiguration.png)You
```
Comment (TamimiGitHub) [inline markdown/sap-aem-int-day-3/sap-aem-int-day-3.md:526]: You have an extra `you` here

**Comment by TamimiGitHub** [comment]
Very minor, but could you please add a `Duration: 0:5:00` under the `## Troubleshooting` section as well

**Comment by ChristianHoltfurth** [comment]
Happy to, but now I'm stuck with the incompatible path issue with someone having used a colon in a link to an image (see Slack).
Can't pull your merge in because of this.

**Comment by TamimiGitHub** [comment]
> Happy to, but now I'm stuck with the incompatible path issue with someone having used a colon in a link to an image (see Slack). Can't pull your merge in because of this.

Fixed the issue and updated this branch with latest

**Pull request merged**